### ISSUE TITLE
Change in formState property is not passed down to state in LoginForm component

### ISSUE THREAD

**Issue by shoetten** [studiointeract/accounts-ui#39]
Hey there,

I noticed that once the LoginForm component is constructed nothing happens when the formState property is changed, through e.g. something like this:

```javascript
FlowRouter.route('/login', {
    name: 'users.login',
    action() {
      mount(MainLayoutCtx, {
        content: () => (<Accounts.ui.LoginForm {...{
          formState: STATES.SIGN_IN,
          signUpPath: '/signup',
        }} />),
      });
    },
  });
```

So when your app has two links in the navbar, one to the signup and one to the login form, the user cannot switch between the two forms using these links.

I think it's simple to fix and will provide a pull request shortly..

**Comment by sessionwire** [comment]
I just discovered this as well.

**Comment by yanickrochon** [comment]
Same here. The problem is [here](https://github.com/studiointeract/accounts-ui/blob/45d2bab633197895b6b68695ee4b0422c450e86e/imports/ui/components/LoginForm.jsx#L22) where the `props.formState` value is completely ignored. This is a trivial change, and before someone tell me to propose a PR... come on!

**Comment by shoetten** [comment]
@sessionwire @yanickrochon You can clone [my fork](https://github.com/shoetten/accounts-ui) directly in the packages directory of your meteor app, while we wait for the #40 (or another solution) to be merged.

**Comment by timbrandin** [comment]
Merged and published in `1.2.1`.
